### PR TITLE
fix: fix nav bar

### PR DESCRIPTION
**Pull request by julienrbrt**
- fix nav bar versioning
- update algolia index to correct index
- set correct baseURL for current version (in addition to other versions). This keeps the same behavior with the current docs, and gives better UX
- temporarily disable redirects as they do not work (should be fixed in a follow-up cc @samricotta)
-  remove chmod from actions and push the script directly with the correct permissions.

### CODE REVIEW

File: docusaurus.config.js [hunk 1]
```diff
@@ -4,16 +4,15 @@
...
 
 /** @type {import('@docusaurus/types').Config} */
 const config = {
   title: "Explore the SDK",
   tagline:
-      "Cosmos SDK is the world's most popular framework for building application-specific blockchains.",
+    "Cosmos SDK is the world's most popular framework for building application-specific blockchains.",
   url: "https://docs.cosmos.network",
-  baseUrl: "/cosmos-sdk-docs",
+  baseUrl: "/cosmos-sdk-docs", // TODO to change on deployed
```
Comment (julienrbrt) [inline docusaurus.config.js:15]:
```suggestion
  baseUrl: "/cosmos-sdk-docs", // TODO to change before live.
```

File: docusaurus.config.js [hunk 2]
```diff
@@ -240,20 +237,20 @@ const config = {
...
+            existingPath.replace("/core", "/develop/advanced"),
+            existingPath.replace("/basics", "/develop/beginner"),
+            existingPath.replace("/intro", "/develop/intro"),
+            existingPath.replace("/architecture", "/build/architecture/"),
+            existingPath.replace("/building-apps", "/build/building-apps"),
             // existingPath.replace('/building-modules', '/build/building-modules'),
-            existingPath.replace('/tooling', '/build/libraries'),
-            existingPath.replace('/migrations', '/build/libraries'),
+            existingPath.replace("/tooling", "/build/libraries"),
+            existingPath.replace("/migrations", "/build/libraries"),
```
Comment (julienrbrt) [inline docusaurus.config.js:247]: I haven't introduced that here, this is why I do not fix it, but this looks wrong.

Comment (samricotta) [reply docusaurus.config.js]: This is fine for removal, Ive added the links in (aligning to the previous way) in the other pr open

**Comment by samricotta** [comment]
tACK 

nb: I dont see what has been added to fix versioning/what is exactly different? - for my own understanding

**Review by samricotta — APPROVED** [review]

**Comment by julienrbrt** [comment]
> tACK
> 
> nb: I dont see what has been added to fix versioning/what is exactly different? - for my own understanding

The current version currently has no version: e.g. /build/building-modules/intro
While non-current do e.g. /0.47/develop/intro/what-is-sdk

Now it is /v0.47/develop/intro/what-is-sdk and the current is /v0.50/build/building-modules/intro, versions are always prepended

**Pull request merged**